Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -58,6 +58,7 @@
 static const float s_kCGGradientOffsetPoint = 1E-45;
 
 enum _CGCoordinateMode : unsigned int { _kCGCoordinateModeDeviceSpace = 0, _kCGCoordinateModeUserSpace };
+enum _CGAntialiasToggle : unsigned int { _kCGToggleDefault = 2, _kCGToggleOff = 0, _kCGToggleOn = 1 };
```

Comment:
I question a separate value for default, shouldn't it be one of the two real options?